server_apps/jenkins/jobs/Check-1-To-Many-ENSDARG-ZFIN/config.xml

```diff
@@ -97,11 +97,5 @@
     </hudson.plugins.emailext.ExtendedEmailPublisher>
   </publishers>
   <buildWrappers>
-    <EnvInjectBuildWrapper plugin="envinject@1.89">
```

Comment:
I thought we need this plugin, don't we?

Reply:
We do but it wasn't working correctly under Jenkins 2.375. Not with docker, at least. I switched it to being injected globally and that worked, so we don't need it set on each individual job.

Reply:
See <globalNodeProperties> in the global config.xml.